provider/resources.go

```diff
@@ -2387,7 +2392,24 @@ func Provider() tfbridge.ProviderInfo {
 			"aws_signer_signing_profile_permission": {Tok: awsResource(signerMod, "SigningProfilePermission")},
 
 			// ServerlessRepository
-			"aws_serverlessrepository_stack": {Tok: awsResource(serverlessRepositoryMod, "Stack")},
+			"aws_serverlessapplicationrepository_cloudformation_stack": {
+				Tok: awsResource(serverlessRepositoryMod, "CloudformationStack"),
```

Comment:
Should it be `CloudFormationStack` as per branding?

Reply:
Great catch! thanks for that - fixing up now